internal/builder/_static/ecs_mappings.json

```diff
@@ -0,0 +1,886 @@
+{
+  "mappings": {
+    "properties": {
+      "@timestamp": {
+        "type": "date",
+        "ignore_malformed": false
+      }
```

Comment:
I used the same json that was built here https://gist.github.com/P1llus/e0de7b3a7824a41a29660e253c6cce6b

I think it could be changed to be a dynamic template as you mentioned. @P1llus do you think in any inconvenient if it is changed ?

I would say there is no problem and it would simplify the code to just manage dynamic templates.

Reply:
For now I've moved that property to a dynamic template as suggested by @jsoriano.

If there is any inconvenient because of that, it could be re-introduced.

Currently, it has been followed the approach to add a prefix (`_embedded_ecs.*`) to all the names of the dynamic templates in the static file, so they can be identified easily. If properties are needed too, it should be added another way (using `_meta` field and lists as it was in a previous change?).